Makefile

```diff
@@ -28,6 +28,18 @@ ifneq ($(SKIP_DOCKER),true)
 		-v $(PROJECT_ROOT):/workspaces/brigade \
 		-w /workspaces/brigade \
 		$(GO_DEV_IMAGE)
+
+	KANIKO_IMAGE := krancour/kaniko:v0.2.0
```

Comment:
This should probably be pushed to `brigadecore`.

Reply:
I think I made a note of that already. I'm going to open an issue about that.

Reply:
https://github.com/brigadecore/brigade/issues/1192